person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
-  def initialize(age:, name: 'Unknown', parent_permission: true)
+  def initialize(age, name: 'Unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
```

Comment:
- [ ] A method for adding rentals should be present, please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more details.